docs/DeveloperGuide.md

```diff
@@ -238,6 +238,17 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Clear Feature
+
+The `clear` feature is used to clear all persons from the displayed list in EmployeeManager.
+
+#### Proposed Implementation
+When the `clear` command is executed, it will call `Model#clearSortedFilteredPersonList`.
+The method will loop through the displayed list and delete each person from the list using the
```

Comment:
Perhaps it would be good to explicitly state that the "list" is referring to the filtered list, not the entire original list?

Reply:
good idea! changed